web/modeltr/connection.py

```diff
@@ -1,37 +1,39 @@
-from pymongo import MongoClient
+# from pymongo import MongoClient
 from contextlib import contextmanager
+import psycopg2
 import logging
 
 DEFAULT_CONNECTION_NAME = 'default'
 
 
 class Connection(object):
     def __enter__(self):
-        self.session = self.client.start_session()
         return self
 
     def __exit__(self, type, value, traceback):
         if traceback is None:
-            self.__logger.debug('Connection successful')
+            self.client.commit()
+            pass
         else:
-            self.__logger.debug('Exception occurred in Connection')
+            self.client.rollback()
+            self.__logger.warn('Exception occurred in Connection, all changes rolled back')
 
-        self.session.end_session()
-
-    # example usage: web.modeltr.connection.Connection(
-    #   host='localhost', authSource='test_database', replicaSet='rs0')
-    # auth source must be present here
     def __init__(self, **kwargs):
         self.__logger = logging.getLogger(__name__)
-        self.client = MongoClient(**kwargs)
-        self.database = kwargs['authSource']
+        for i in range(6):
```

Comment:
This would be better to have in config file.

Reply:
done